org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/LogHandler.java

```diff
@@ -68,6 +68,8 @@ private int getLogLevelMask(String logLevel) {
 			return -1;
 		case "ERROR":
 			return IStatus.ERROR;
+		case "INFO"	:
```

Comment:
I would just remove the whitespace between "INFO" and ':' to match everything else.

Reply:
Done!